src/resolvers.js

```diff
@@ -29,6 +29,93 @@ export async function fetchThemeJson() {
 	}
 }
 
+export async function fetchThemeStyleData() {
+	const fetchOptions = {
+		path: '/create-block-theme/v1/get-style-data',
+		method: 'GET',
+		headers: {
+			'Content-Type': 'application/json',
+		},
+	};
+
+	try {
+		const response = await apiFetch( fetchOptions );
+		if ( ! response?.data || 'SUCCESS' !== response?.status ) {
+			throw new Error(
+				`Failed to fetch style data: ${
+					response?.message || response?.status
+				}`
+			);
+		}
+		return response?.data;
+	} catch ( e ) {
+		// @todo: handle error
```

Comment:
Could we handle this error?

Reply:
Not there, in the resolvers.  The things calling the resolvers are handling the error.  But that todo would prevent the errors from being caught so I removed it (and standardized all of the functions while I was at it).

Reply:
In that case, I think the catch should either throw, or not be wrapped in a try catch statement at all.